### PR TITLE
Upgrade dependencies with available patch and minor upgrades

### PR DESCRIPTION
**Pull request by francoislehoux-okta**
This is a first pass of multiple like this. This will slowly get us back into a state where everything is on the latest version.

## Description:
- **What's changed?** 
- axios
- semver
- include-media
- jshint
- moment
- moment-timezone
- query-string
- vue-select

### Resolves:
* [OKTA-490911](https://oktainc.atlassian.net/browse/OKTA-490911)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
   "author": "Brian Retterer <brian.retterer@okta.com>",
   "license": "Apache-2.0",
   "devDependencies": {
-    "axios": "^0.21.2",
+    "axios": "^0.26.1",
```
Comment (shawyu-okta) [inline package.json:34]: I looked over the [changelog](https://github.com/axios/axios/blob/master/CHANGELOG.md) and it seems like the breaking changes do not affect us, but worth grepping through the code and ensuring our uses don't overlap with any breaking changes.

File: packages/@okta/vuepress-theme-prose/package.json [hunk 2]
```diff
@@ -25,17 +25,17 @@
...
     "css-to-json": "^2.1.0",
-    "include-media": "^1.4.9",
-    "jshint": "^2.12.0",
+    "include-media": "^1.4.10",
+    "jshint": "^2.13.4",
     "jwt-decode": "^3.1.2",
-    "moment": "^2.29.2",
-    "moment-timezone": "^0.5.32",
+    "moment": "^2.29.3",
+    "moment-timezone": "^0.5.34",
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/package.json:36]: This change is just updating TZ data so we should be good here

File: packages/@okta/vuepress-theme-prose/package.json [hunk 3]
```diff
@@ -25,17 +25,17 @@
...
     "portal-vue": "^2.1.7",
-    "query-string": "^6.13.8",
+    "query-string": "^7.1.1",
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/package.json:38]: https://github.com/sindresorhus/query-string/releases shows breaking change to `.stringify` but we don't use it. Should be fine.

**Review by shawyu-okta — APPROVED** [review]
I marked any semver changes that could contain breaking changes. I think `axios` might be the only one we need to double check but I do think it's probably also safe.

**Comment by francoislehoux-okta** [comment]
> I marked any semver changes that could contain breaking changes. I think `axios` might be the only one we need to double check but I do think it's probably also safe.

I'll reverify `axios` but I've already done a first compatibility check

**Comment by francoislehoux-okta** [comment]
Did a manual verification. Token retrieval and signup still work: https://625f1508b9cefd219ba8ca12--reverent-murdock-829d24.netlify.app/ @shawyu-okta

**Comment by shawyu-okta** [comment]
Thanks for checking @francoislehoux-okta 

:shipit:

**Pull request merged**